src/OWML.ModHelper.Input/ModCommandListener.cs

```diff
@@ -7,34 +7,34 @@ namespace OWML.ModHelper.Input
 {
 	public class ModCommandListener : MonoBehaviour
 	{
-		//private readonly HashSet<SingleAxisCommand> _commands = new();
+        private readonly HashSet<IInputCommands> _commands = new();
```

Comment:
we use tabs everywhere else in OWML. You can configure it in VS

Reply:
done